lib/services/role.go

```diff
@@ -111,33 +111,25 @@ func RoleNameForCertAuthority(name string) string {
 // is not explicitly assigned (this role applies to all users in OSS version).
 func NewAdminRole() types.Role {
 	adminRules := getExtendedAdminUserRules(modules.GetModules().Features())
-	role := &types.RoleV3{
-		Kind:    types.KindRole,
-		Version: types.V3,
-		Metadata: types.Metadata{
-			Name:      teleport.AdminRoleName,
-			Namespace: apidefaults.Namespace,
+	role, _ := types.NewRole(teleport.AdminRoleName, types.RoleSpecV3{
```

Comment:
here and below: return the error

Reply:
Since a non nil error here implies a bug in and of itself, I think it would be better to add a separate test for these basic roles, as suggested in https://github.com/gravitational/teleport/issues/7235.